src/scripts/main.js

```diff
@@ -357,4 +357,26 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+for (const person of people) {
+  const createdTr = document.createElement('tr');
+  const { sex, born, died } = person;
+  const tableRowData = {
+    name: person.name,
+    gender: sex,
```

Comment:
compare your table with examples in task description.
in table should be rendered Male or Female strings.
Make sure all test passed (provide screenshot with passed tests) and add demo link to pr description